components/logo/logo.vue

```diff
@@ -0,0 +1,18 @@
+<template>
+  <img :src="src" :srcset="srcset" class="mesg" />
+</template>
+
+<script>
+export default {
```

Comment:
I don't see the point of this component if it's just an image. Either we don't do this component or we can make this component smart.

This component could actually load the pictures and then we could use it with the following
```
<icon name="ethereum" />
```

Reply:
Done.